src/bert/utils.py

```diff
@@ -92,7 +93,18 @@ def forward(self, sent: Dict[str, torch.LongTensor],
         if self.embeddings_mode != "only":
             # encoded_layers is a list of layer activations, each of which is
             # <float32> [batch_size, seq_len, output_dim]
-            encoded_layers, _ = self.model(ids, token_type_ids=torch.zeros_like(ids),
+            if pair_task:
```

Comment:
Split this block into a helper function with a detailed docstring? (maybe `get_token_type_ids()`)